src/helpers/hotkeysManager.js

```diff
@@ -251,6 +253,26 @@ WebViewer(...)
           core.setCurrentPage(currPageNumber + 1);
         }
       },
+      up: e => {
+        // do not call preventDefault else it will prevent scrolling
+        const scrollViewElement = core.getScrollViewElement();
+        const { scrollTop } = scrollViewElement;
+        const reachedTop = scrollTop === 0;
+        const currPageNumber = core.getCurrentPage();
+        if ((e.key === 'ArrowUp' || e.which === 38) && reachedTop && currPageNumber > 1) {
```

Comment:
I don't think we need to check the `key` and `which` here since the library handles this for us. The library should call this handler only if the `key` is `ArrowUp`

Reply:
addressed. thanks for pointing that out. i must have blindly copied and pasted.